### DISCUSSION TITLE
Heroku deployment [Jan 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4807] [Discord Threads]
> 🙋‍♂️ **jtviolet**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Should work. Just that the free experimental tier's doesn't exist anymore in Heroku starting from Nov 28th 2022.  
#  
> 🙋‍♂️ **jtviolet**  
#### Hmm, unfortunately it doesn’t seem to be working.  
#  
> 👨‍🦲 **dstala**  
#### Can you share details about steps that you tried? Your configurations?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1048450345216524308)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
